### DISCUSSION TITLE
Remove unused tag feature

### DISCUSSION THREAD

**Discussion by freddidierRTE** [opfab/operatorfabric-core#2187] [Ideas]
The idea is to remove part of tags features which is not used and using a component which is not compatible with last version of angular (13) 

**Remove tag search in feed**

![image](https://user-images.githubusercontent.com/58251627/144852914-fa8f6743-d1b1-41cf-88fb-f4b541a65512.png)

**Remove tag preference in settings screen** 

![image](https://user-images.githubusercontent.com/58251627/144853028-4ff70cbc-7120-4733-99ac-8add12e5a117.png)

**But **KEEP TAGS  list in archives and logging screen** for searching cards** 

![image](https://user-images.githubusercontent.com/58251627/144853127-9ce437a1-f5bf-492d-8ce5-df271dc86933.png)

**Comment by JeroenGommans** [comment]
It is fine with me. Great you are able to keep the tags in the archives screen!

**Comment by OlivierVoron** [comment]
we are Ok with the removal of the feature (filtering tag in Feed). However we will make, in the following days, some other propositions to help an end-user filtering multiple notifications in the feed by one click  and focusing for example to a specific process

**Comment by freddidierRTE** [comment]
Ok will be done via issue #2197
